airbyte-webapp/build.gradle

```diff
@@ -5,98 +5,112 @@ plugins {
 
 def nodeVersion = System.getenv('NODE_VERSION') ?: '16.15.1'
 
+// This array should contain a path to all configs that are common to most build tasks and
+// might affect them (i.e. if any of those files change we want to rerun most tasks)
+def commonConfigs = [
```

Comment:
:information_source: Extract common configuration, that might affect all tasks to require a rerun out into a variable.